angular-chess/src/app/services/board-theming.service.ts

```diff
@@ -0,0 +1,32 @@
+import { ElementRef, Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+
+export interface BoardThemeConfig {
+  darkField: string;
+  lightField: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BoardThemingService {
+
+  private readonly woodTheme: BoardThemeConfig = {
+    darkField: 'url(/board_field_dark.png)',
+    lightField: 'url(/board_field_light.png)'
+  };
+
+  private readonly brownTheme: BoardThemeConfig = {
+    darkField: '#b58863',
+    lightField: '#f0d9b5',
+  };
```

Comment:
The contrast to the white pieces is not so good. Here is a suggestion for a different coloring of the fields:
```suggestion
  private readonly brownTheme: BoardThemeConfig = {
    darkField: '#882d17',
    lightField: '#c19a6b',
  };
```
It would look like this: 
![image](https://user-images.githubusercontent.com/25728708/159579809-47097821-36f0-414c-bdb0-e5bb949b8c41.png)